tests/lib/jss.js

```diff
@@ -435,5 +435,41 @@ describe('JSS', function () {
             ].join('\n');
             expect(result).to.equal(expected);
         });
+        it('should combine pseudo selectors if ie options is off', function () {
```

Comment:
This test seems to test both cases (on and off), so perhaps it should be broken up into two tests to be more clear what we're testing?